src/pages/partners/affiliate-ib/_deriv-ib-programme.tsx

```diff
@@ -131,7 +134,61 @@ const SubtitleHeader = styled(Header)`
     }
 `
 
+type ItemProps = {
+    active_tab: 'mt5' | 'x'
```

Comment:
I guess you can make this type reusable. As you are using the same in line `181` so isn’t it good to make it reusable 👀  

Thanks @eduard-deriv 🙏

Reply:
yap, forgot to clean up, thanks